Src/base/application/ApplicationManagerService.cpp

```diff
@@ -349,7 +349,7 @@ static bool servicecallback_close( LSHandle* lshandle, LSMessage *message,
 	processid = json_object_object_get(root,"processId");
 	if(processid) {
 		qint64 processId = (qint64) atol(json_object_get_string(processid));
-		if (processid > 0) {
```

Comment:
actually, it looks like a typo here: what the dev wanted to test was "processId", not "processid". Very bad choice of variable name here !

Reply:
good that gcc-11 finally help you catch that :)